Phone-Number-Validator/readme.md

```diff
@@ -0,0 +1,10 @@
+# Phone Number Validator
+
+![](https://github.com/SANTHOSH17-DOT/Hacktoberfest-Projects/blob/phoneNumValidation-SANTHOSH17-DOT/Phone-Number-Validator/project.gif)
```

Comment:
Gif is not showing

Reply:
Could you check it again??
It seems to be working fine.